### PR TITLE
Avoid so much duplicate code

### PR DESCRIPTION
**Pull request by techcobweb**
Signed-off-by: Mike Cobbett <77053+techcobweb@users.noreply.github.com>

- Lots of duplicate code in a deprecated constructor refactored into separate method.
- Left alone the initialisation of the final variables.

### CODE REVIEW

File: galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zos3270.manager/src/main/java/dev/galasa/zos3270/spi/Zos3270TerminalImpl.java [hunk 1]
```diff
@@ -153,36 +136,51 @@ public Zos3270TerminalImpl(String id, String host, int port, boolean tls, IFrame
...
                 // *** Register the terminal to the UI which will own the terminal view
                 HttpURLConnection connection = (HttpURLConnection) propLiveTerminalUrl.openConnection();
                 connection.setRequestMethod("HEAD");
-                connection.addRequestProperty("zos3270-runid", this.runId);
-                connection.addRequestProperty("zos3270-terminalid", this.terminalId);
+                connection.addRequestProperty("zos3270-runid", runId);
+                connection.addRequestProperty("zos3270-terminalid", terminalId);
                 connection.setDoInput(true);
                 connection.setDoOutput(false);
                 connection.connect();
```
Comment (techcobweb) [inline galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zos3270.manager/src/main/java/dev/galasa/zos3270/spi/Zos3270TerminalImpl.java:170]: I guess we are assuming that garbage collection of the connection will close the connection, and we don't have to do it explicitly ?

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Pull request closed without merging**